cmd/test/republic.go

```diff
@@ -0,0 +1,249 @@
+package test
```

Comment:
We can get rid of this test folder as we have local integration tests in the dark-node folder.
They are almost doing the same thing.

Reply:
Ok